src/gui/profilergraph.cpp

```diff
@@ -95,12 +96,12 @@ void ProfilerGraph::draw(s32 x_left, s32 y_bottom, video::IVideoDriver *driver,
 
 		s32 texth = 15;
 		char buf[10];
-		snprintf(buf, 10, "%.3g", show_max);
+		porting::mt_snprintf(buf, 10, "%.3g", show_max);
```

Comment:
Looks like a superfluous change. The size of the resulting string is known and there are no `%1$s` replacements.

Reply:
I think the point here is that you would always use `mt_snprintf` so you can be sure everything works as you expect in all possible cases.